app/views/partials/header.ejs

```diff
@@ -0,0 +1,44 @@
+<% const { show = true, menu = true, home = false, fullscreen = false } = locals; %>
+
+<!DOCTYPE html>
+<html lang="<%= locals.htmlLang %>">
+<head>
+<%- include('partials/head') -%>
+</head>
+<body class="flex flex-col">
+  <div class="static w-full top-0 left-0 right-0">
+    <div class="hidden lg:block">
+      <%- include('partials/banner', {
+        start: __('NB!'),
+        message: __('This page is currently not affiliated with the authorities in any way.'),
+        link: urls.privacyPolicy,
```

Comment:
Are `urls` available here when you don't `destructure` it from locals?

Reply:
Yeah, locals are per definition available in the templates, but constructing them is nice because you can give them default variables.

Reply:
Ah, didn't know